### PR TITLE
Updates to SES7 deployment process

### PR DESCRIPTION
**Pull request by tbazant**
This update includes:
* several general doc updates/corrections
* deployment of MDS, RGW, and NFS services

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -1089,6 +1116,71 @@ ses-min4
      </listitem>
     </itemizedlist>
    </sect3>
+   <sect3 xml:id="deploy-cephadm-day2-service-mds">
+    <title>Deploy &mds;s</title>
+    <para>
+     To use &cephfs;, you need to deploy at least one &mds; (MDS) service. You
+     can either deploy MDS's without &cephfs;, or create &cephfs; and deploy an
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1123]:
```suggestion
     can either deploy MDSs without &cephfs;, or create &cephfs; and deploy an
```

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -1089,6 +1116,71 @@ ses-min4
...
+     MDS automatically.
+    </para>
+    <para>
+     To deploy &mds; without &cephfs;, run:
+    </para>
+<screen>&prompt.cephuser;ceph orch apply mds <replaceable>CEPH_FS_NAME</replaceable> <replaceable>HOST1</replaceable>,<replaceable>HOST2</replaceable>,...</screen>
+    <para>
+     To create &cephfs; and deploy &mds; automatically, run:
```
Comment (lproven) [inline xml/deploy_cephadm.xml:1131]: I am not sure -- is this one MDS or several? 

If one, then either "deploy _the_ MDS" or "deploy _an_ MDS".

File: xml/deploy_intro.xml [hunk 3]
```diff
@@ -14,8 +14,23 @@
   </dm:docmanager>
  </info>
  <para>
-  INTRO
+  Since &productname; &productnumber;, &ceph; services are deployed as
```
Comment (lproven) [inline xml/deploy_intro.xml:17]: Should this be a fixed version number, rather than an entity?

File: xml/deploy_intro.xml [hunk 4]
```diff
@@ -14,8 +14,23 @@
...
+ <itemizedlist>
+  <listitem>
+   <para>
+    By using the &ceph; module. Find more details in
+    <xref linkend="deploy-cephadm"/>.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    On top of &casp; cluster. Find more details in
```
Comment (lproven) [inline xml/deploy_intro.xml:29]:
```suggestion
    On top of a &casp; cluster. Find more details in
```

File: xml/deploy_cephadm.xml (hunk 2 above)
Comment (tbazant) [inline xml/deploy_cephadm.xml:1131]:
```suggestion
     To create &cephfs; and deploy the &mds; automatically, run:
```

File: xml/deploy_intro.xml (hunk 3 above)
Comment (tbazant) [inline xml/deploy_intro.xml:17]:
```suggestion
  Since &productname; 7, &ceph; services are deployed as
```

File: xml/deploy_cephadm.xml [hunk 5]
```diff
@@ -1089,6 +1126,71 @@ ses-min4
...
+   </sect3>
+   <sect3 xml:id="deploy-cephadm-day2-service-nfs">
+    <title>Deploy &ganesha;</title>
+    <para>
+     &cephadm; deploys &ganesha; using a pre-defined &rados; pool and an
+     optional namespace:
+    </para>
+<screen>&prompt.cephuser;ceph orch apply nfs <replaceable>SERVICE_ID</replaceable> \
+ <replaceable>POOL_NAME</replaceable> <replaceable>NAMESPACE</replaceable> \
+ --placement="<replaceable>NUM_OF_DAEMONS</replaceable> <replaceable>HOST1</replaceable> <replaceable>HOST2</replaceable>,..."</screen>
```
Comment (tbazant) [inline xml/deploy_cephadm.xml:1192]:
```suggestion
 --placement="<replaceable>HOST1</replaceable>,<replaceable>HOST2</replaceable>,..."</screen>
```

**Pull request merged**